py_gtfs_rt_ingestion/ingest.py

```diff
@@ -82,24 +82,29 @@ def validate_environment() -> None:
         )
 
 
-@schedule.repeat(schedule.every().hour.at(":05"))
-def batch_and_ingest() -> None:
+@schedule.repeat(schedule.every(5).minutes)
```

Comment:
i made this more frequent to find errors easier. maybe its ok to change it back now?